rct229/rulesets/ashrae9012019/section19/section19rule36.py

```diff
@@ -0,0 +1,94 @@
+
+ENERGY_RECOVERY = schema_enums["EnergyRecoveryOptions"]
+REQ_SENSIBLE_EFFECTIVENESS = 0.5
+REQ_LATENT_EFFECTIVENESS = 0.5
+
+
+class Section19Rule36(RuleDefinitionListIndexedBase):
+    """Rule 36 of ASHRAE 90.1-2019 Appendix G Section 19 (HVAC - General)"""
+
+    def __init__(self):
+        super(Section19Rule36, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section19Rule36.HVACRule(),
+            index_rmr="baseline",
+            id="19-36",
+            description="Baseline systems required to model energy recovery per G3.1.2.10 shall be modeled with a 50% enthalpy recovery ratio.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1-5(a)5 Building Envelope Modeling Requirements for the Proposed design",
+            is_primary_rule=False,
+            list_path="ruleset_model_descriptions[0].buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    class HVACRule(PartialRuleDefinition):
+        def __init__(self):
+            super(Section19Rule36.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                required_fields={"$": ["fan_system"]},
+            )
+
+        def get_calc_vals(self, context, data=None):
+            fan_sys_b = context.baseline
+
+            sensible_eff_b = find_one(
+                "$.air_energy_recovery.design_sensible_effectiveness", fan_sys_b
+            )
+            latent_eff_b = find_one(
+                "$.air_energy_recovery.design_latent_effectiveness", fan_sys_b
+            )
+            ERV_OA_airflow_b = find_one(
+                "$.air_energy_recovery.outside_air_flow", fan_sys_b
+            )
+            ERV_EA_airflow_b = find_one(
+                "$.air_energy_recovery.exhaust_air_flow", fan_sys_b
+            )
+            hvac_min_oa_flow_b = find_one("$.minimum_outdoor_airflow", fan_sys_b)
+
+            return {
+                "sensible_eff_b": sensible_eff_b,
+                "latent_eff_b": latent_eff_b,
+                "ERV_OA_airflow_b": CalcQ("air_flow_rate", ERV_OA_airflow_b),
+                "ERV_EA_airflow_b": CalcQ("air_flow_rate", ERV_EA_airflow_b),
+                "hvac_min_oa_flow_b": CalcQ("air_flow_rate", hvac_min_oa_flow_b),
+            }
+
+        def applicability_check(self, context, calc_vals, data):
+            fan_sys_b = context.baseline
+
+            sensible_eff_b = calc_vals["sensible_eff_b"]
+            latent_eff_b = calc_vals["latent_eff_b"]
+            ERV_OA_airflow_b = calc_vals["ERV_OA_airflow_b"]
+            ERV_EA_airflow_b = calc_vals["ERV_EA_airflow_b"]
+            hvac_min_oa_flow_b = calc_vals["hvac_min_oa_flow_b"]
+
+            return (
+                fan_sys_b.get("air_energy_recovery") is not None
+                and fan_sys_b["air_energy_recovery"] != ENERGY_RECOVERY.NONE
```

Comment:
What is the difference between Ln73 and Ln74? @yunjoonjung-PNNL

Reply:
Line 73 checks whether the ```air_energy_recovery``` exists or not. And, Line 84 checks whether the ```air_energy_recovery``` type is ```NONE``` or not. More specifically, if you find the ```EnergyRecoveryOptions``` in the schema file, you'll find there is the ```NONE``` which is one of the enumeration options.

Please let me know if this isn't straightforward. I'll let you know about this again.